repos/rust-lang/gcc.toml

```diff
@@ -0,0 +1,11 @@
+org = "rust-lang"
+name = "gcc"
+description = "GCC fork with libgccjit changes needed by rustc_codegen_gcc"
+bots = []
+
+[access.teams]
+compiler = "write"
+
+[access.individuals]
+antoyo = "maintain"
```

Comment:
Given that this "combo" of individuals :) Is used for several repos now, it would probably make sense to create a small team/wg for this.